pkg/client/cache/listers.go

```diff
@@ -77,37 +77,37 @@ type storePodsNamespacer struct {
 // the store; there may have been some filtering that already happened before
 // that.
 func (s storePodsNamespacer) List(selector labels.Selector) (pods api.PodList, err error) {
-	list := api.PodList{}
 	if s.namespace == api.NamespaceAll {
 		for _, m := range s.indexer.List() {
 			pod := m.(*api.Pod)
 			if selector.Matches(labels.Set(pod.Labels)) {
-				list.Items = append(list.Items, *pod)
+				pods.Items = append(pods.Items, *pod)
 			}
 		}
-		return list, nil
+		return
```

Comment:
`return pods, err`

I don't care myself, but some people find it less confusing.